git/odb/storer.go

```diff
@@ -0,0 +1,16 @@
+package odb
+
+import "io"
+
+// Storer implements a storage engine for reading, writing, and creating
+// io.ReadWriters that can store information about loose objects
+type Storer interface {
```

Comment:
This doesn't need to be exported, since it's only used on a private `ObjectDatabase` property.